web/src/components/app.tsx

```diff
@@ -332,10 +339,28 @@ class App extends React.Component {
 
     Sentry.init({
       dsn:
-        'https://4a940c31e4e14d8fa6984e919a56b9fa@sentry.prod.mozaws.net/491',
+        SENTRY_FE_DSN,
       environment: isProduction() ? 'prod' : 'stage',
       release: process.env.GIT_COMMIT_SHA || null,
     });
+
+    if (!isProduction()) {
+      console.log('disabling analytics on non production');
+    } else if (!doNotTrack()) {
+      console.log('do not track header set, disabling analytics');
```

Comment:
Should this be `else if (doNotTrack()) {` ?

Reply:
LOL duh yes, thank god for code reviews. I actually decided to remove the separate console messages entirely, I realized that since this check is already being done in `index_template` for Google Analytics, repeating the check here would simply result in redundant messaging.